RELEASE.md

```diff
@@ -19,9 +19,17 @@ documentation].
 - Write the [changelog](CHANGELOG.md) with user-visible API changes. This must
   go through the regular PR review process and get merged into the `master` branch.
   Approval of the PR indicates community consensus for a new release.
-- Once the above PR is merged, the author must publish a new Git tag. This can
-  be done using the `git` CLI or Github's [release][release]
-  page. This step can be performed only by [Gateway API maintainers][gateway-api-team].
+
+The following steps must be done by one of the [Gateway API maintainers][gateway-api-team]:
+- If you are making a new major or minor release, cut a `release-major.minor` branch that we
+  can tag things in as needed.
+- Check out the `release-major.minor` branch locally
+- Update `pkg/generator/main.go` with the new semver tag and any updates to the API review URL.
+- Run the following commnand `BASE_REF=vmajor.minor.patch make generate` which will update generated docs
```

Comment:
Do you need to do this for every patch release too?

Reply:
Yes, good point. I'll try and make this clearer.

Reply:
Although the individual bullet points look good now, I think this still needs to be split up by subheading. So instead of everything being under "For a major or minor release", you could have ~3 sections:

- Major or minor release
- Patch release
- All release

There's not much unique for a patch release, but should at least mention that every patch release needs to originate from a release-major.minor branch.